civihr_default_theme/templates/view/views-view-field--emergency-contacts--non-dependant-emergency-contact--emergency_contact_deletion.tpl.php

```diff
@@ -13,10 +15,12 @@
           <p>This cannot be undone</p>
         </div>
         <div class="modal-footer">
-          <button type="button" class="btn btn-link text-uppercase">Cancel</button>
-          <form action="/action_page.php?id=<?php print $row->id ?>" method="get">
-            <input class="btn btn-danger" type="submit" value="Confirm">
-          </form>
+          <button type="button" class="btn btn-link text-uppercase">
+            Cancel
+          </button>
+          <button onclick="Drupal.civihr_theme.deleteEmergencyContact(<?php print $row->id ?>); location.reload();">
```

Comment:
I am not sure this the good practice to call two functions on `onClick` event. We should move such code in separate function I guess.

In fact why can't we do as following?
```js
Drupal.civihr_theme.deleteEmergencyContact =  function (id) {
   CRM.api3('contact', 'deleteemergencycontact', { 'id': id });
   location.reload();
}
```